README.md

```diff
@@ -96,22 +99,66 @@ public class YourUserContext : IUserContext
 }
 ```
 
-3. **Use in Your Services**:
+3. **Implement JSON Converter** (required):
+
+You must implement `IJsonConverter` using your preferred JSON serializer. Here are examples for popular serializers:
+
+**Using Newtonsoft.Json:**
+```csharp
+using HybridMicroOrm.Contracts;
+using Newtonsoft.Json;
+
+public class NewtonsoftJsonConverter : IJsonConverter
+{
+    public string Serialize(object value)
+    {
+        return JsonConvert.SerializeObject(value);
+    }
+
+    public T Deserialize<T>(string json)
+    {
+        return JsonConvert.DeserializeObject<T>(json)!;
+    }
+}
+```
+
+**Using System.Text.Json:**
+```csharp
+using HybridMicroOrm.Contracts;
+using System.Text.Json;
+
+public class SystemTextJsonConverter : IJsonConverter
+{
+    public string Serialize(object value)
+    {
+        return JsonSerializer.Serialize(value);
+    }
+
+    public T Deserialize<T>(string json)
+    {
+        return JsonSerializer.Deserialize<T>(json)!;
+    }
+}
+```
+
+4. **Use in Your Services**:
 
 ```csharp
 public class CustomerService
 {
     private readonly IHybridMicroOrm _orm;
+    private readonly IJsonConverter _jsonConverter;
 
-    public CustomerService(IHybridMicroOrm orm)
+    public CustomerService(IHybridMicroOrm orm, IJsonConverter jsonConverter)
```

Comment:
The customer service shouldnt need the IJsonConverter as the work is done within the orm using the provided registration of the IJsonConverter.